docs/docsite/rst/user_guide/playbooks_filters_ipaddr.rst

```diff
@@ -434,6 +434,11 @@ To find the next nth usable IP address within a range, use ``next_nth_usable``::
 
 In this example, ``next_nth_usable`` returns the second usable IP address for the given IP range.
 
+To find the peer IP address for a point to point link, use ``peer``::
+
+    # {{ '192.168.122.1/31' | ipaddr('peer') }}
+    192.168.122.0
+
```

Comment:
This is added under the 'Finding IP addresses within a range' heading but isn't really doing that. Consider creating a "Finding the peer of a point-to-point link' heading, or adding this under the 'Basic queries' heading.

Reply:
I can. However, this a /31 is a range and the peer address is contained in the range. It's really finding the "other" IP address of a range. Do you still want me to create a special heading for it?

Reply:
ah no. That's for clarifying. LGTM on the docs side